### DISCUSSION TITLE
For a Remix app, should @vanilla-extract/css be a dependency or devDependency?

### DISCUSSION THREAD

**Discussion by ryanwoodcox** [vanilla-extract-css/vanilla-extract#1358] [Q&A] [answered]
For a Remix app, should @vanilla-extract/css be a dependency or devDependency?

Sources of confusion:

1. Install as a dependency:
https://vanilla-extract.style/documentation/getting-started

2. Install as a devDependency:
https://remix.run/docs/en/main/styling/vanilla-extract

3. Install as a dependency:
https://github.com/vanilla-extract-css/vanilla-extract/blob/4a54c17f2b407fba02a238690a443447c67e5475/examples/remix/package.json#L15

**Comment by askoufis** [comment, marked as answer]
I think most/all bundlers are smart enough to realize that `@vanilla-extract/css` isn't used at runtime, so in practice it doesn't really matter. However, the package is ultimately used at build time, so for consistency's sake it should be classified as a `devDependency`.
